pkg/kubelet/pluginmanager/operationexecutor/operation_executor.go

```diff
@@ -95,10 +97,11 @@ func (oe *operationExecutor) IsOperationPending(socketPath string) bool {
 func (oe *operationExecutor) RegisterPlugin(
 	socketPath string,
 	timestamp time.Time,
+	UUID types.UID,
 	pluginHandlers map[string]cache.PluginHandler,
 	actualStateOfWorld ActualStateOfWorldUpdater) error {
 	generatedOperation :=
-		oe.operationGenerator.GenerateRegisterPluginFunc(socketPath, timestamp, pluginHandlers, actualStateOfWorld)
+		oe.operationGenerator.GenerateRegisterPluginFunc(socketPath, timestamp, UUID, pluginHandlers, actualStateOfWorld)
```

Comment:
@claudiubelu Can you explain why do we want to keep timestamp and pass it around if we switch to using UUID? Do we want to keep using timestamp on non-windows platforms?

Reply:
Sure. The ``oe.operationGenerator.GenerateRegisterPluginFunc`` will call ``actualStateOfWorldUpdater.AddPlugin``, with a ``PluginInfo`` that will have a Timestamp set [1]. The ``PluginInfo``'s timestamp could still be useful information.

[1] https://github.com/kubernetes/kubernetes/blob/f97d14c6c88e92cb505f8a9147294705a93247e3/pkg/kubelet/pluginmanager/operationexecutor/operation_generator.go#L115

Reply:
> The PluginInfo's timestamp could still be useful information.

How it could be useful if it's not used anywhere? I'd suggest to remove it to make an API cleaner.

Reply:
Done